pom.xml

```diff
@@ -143,16 +143,16 @@
 
         <!-- Dependency versions -->
         <slf4jVersion>1.6.4</slf4jVersion>
-        <luceneVersion>5.3.0</luceneVersion>
+        <luceneVersion>5.3.1</luceneVersion>
```

Comment:
Whenever a PR is sent to WF to incorporate a HS version with this change,
that PR needs to update the WF Lucene modules, too. Just a reminder :)